controllers/datafeed_controller.py

```diff
@@ -107,9 +107,20 @@ class FMSAPIEventAlliancesEnqueue(webapp.RequestHandler):
     Handles enqueing getting alliances from the FMS API
     """
     def get(self, when):
+        events = skipped_events = []
         if when == "now":
             events = EventHelper.getEventsWithinADay()
             events = filter(lambda e: e.official, events)
+
+            # Check if we should do a full query, or temporarily skip events
+            # that are not ending today. (All events will be checked on
+            # half-hour intervals, and events ending today whenever the cron
+            # job runs.)
+            minutes_from_full = (((int(time.time()) / 60) + 15) % 30) - 15
```

Comment:
Yeah, that is what that logic is doing. I realize that line is not very clear. I wasn't sure exactly when the cronjobs would run (if they were exactly at XX:00 and XX:30 or slightly off) so I came up with that to let it run anytime within a 5-minute interval.

Two cronjobs would probably be much clearer and more reliable though. Any ideas for what the URLs should be for each endpoint?